tsconfig.json

```diff
@@ -33,7 +33,8 @@
       "#models/*": ["src/app/models/*"],
       "#testHelpers/*": ["src/testHelpers/*"],
       "#pages/*": ["src/app/pages/*"],
-      "#server/*": ["src/server/*"]
+      "#server/*": ["src/server/*"],
+      "react": ["./node_modules/@types/react"]
```

Comment:
Seems to have crept in at some stage with Next. Without this, it causes type checking errors when building the Next app. The Next app inherits this `tsconfig` and extends it.